CredentialProvider.Microsoft/Resources.resx

```diff
@@ -127,13 +127,13 @@
     <value>Failed to acquire session token: {0}</value>
   </data>
   <data name="AdalDeviceFlowMessage" xml:space="preserve">
-    <value>ATTENTION: User interaction required. 
```

Comment:
I'm going to assume these changes are some sort of line ending normalization or something

Reply:
Yup, that got added after I applied the commit suggestion. Should I revert it?